dev/manual_tests/macos/Runner/Configs/AppInfo.xcconfig

```diff
@@ -0,0 +1,14 @@
+// Application-level settings for the Runner target.
+//
+// This may be replaced with something auto-generated from metadata (e.g., pubspec.yaml) in the
+// future. If not, the values below would default to using the project name when this becomes a
+// 'flutter create' template.
+
+// The application's name. By default this is also the title of the Flutter window.
+PRODUCT_NAME = manual_tests
+
+// The application's bundle identifier
+PRODUCT_BUNDLE_IDENTIFIER = com.example.manualTests
```

Comment:
What do we use as a bundle ID on iOS for these tests? Not a big deal either way since it's test, but if we use a flutter.io (or flutter.dev) ID there we probably should here too.

Reply:
I changed it to io.flutter.examples, as the iOS one uses. (although it uses io.flutter.examples.manual_tests, not manualTests).